File: app/src/main/java/com/shindra/ScheduleActivity.java
package com.shindra;

import android.os.Bundle;
import android.util.Log;

import androidx.appcompat.app.AppCompatActivity;

public class ScheduleActivity extends AppCompatActivity {
    //Fragment required

    Bundle b;
    public String nameLine;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        nameLine = getIntent().getStringExtra("keyNameLine");
        Log.d("schedule", "ScheduleActivity demarree, ligne demandee:" + nameLine);
    }
}

File: app/src/main/java/com/shindra/ScheduleActivity.kt
package com.shindra

import android.os.Bundle
import android.util.Log
import androidx.appcompat.app.AppCompatActivity

class SchedulesActivity : AppCompatActivity() {
    private val logTag = "TramsSchedulesActivity"

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.schedule_activity)
        val fragment = supportFragmentManager.findFragmentById(R.id.stopfragment) as StopFragment

    }
}